src/cryptsetup/cryptsetup.c

```diff
@@ -2291,6 +2310,11 @@ static int run(int argc, char *argv[]) {
                         key_file = NULL;
                 }
 
+                /* For now and for compatibility, if the user explicitly configured FIDO2 support and we do
+                 * not read FIDO2 metadata off the LUKS2 header, default to the systemd 248 logic, where we
+                 * use PIN + UP when needed, and do not configure UV at all. */
+                arg_fido2_manual_flags = FIDO2ENROLL_PIN_IF_NEEDED | FIDO2ENROLL_UP_IF_NEEDED | FIDO2ENROLL_UV_OMIT;
```

Comment:
hmm, why set this here, instead of at the top of the file where you declare them?

Reply:
no functional reason, just the comment breaking up the list of arg variables didn't "look right" to my OCD :smile: